src/gui/guiSkin.h

```diff
@@ -2,8 +2,7 @@
 // This file is part of the "Irrlicht Engine".
 // For conditions of distribution and use, see copyright notice in irrlicht.h
 
-#ifndef __GUI_SKIN_H_INCLUDED__
-#define __GUI_SKIN_H_INCLUDED__
+#pragma once

```

Comment:
This is also a modified file from Irrlicht, so please keep the include guard.

Reply:
Should be fixed now.